### DISCUSSION TITLE
Using Theming from v10

### DISCUSSION THREAD

**Discussion by LeoRedin** [infinitered/ignite#2685] [Q&A]
Hey folks! I'm starting a new project and I've been using this template for years now. I know the PR for theming hasn't been merged yet but would I be able to use it now? I could just grab the new approach and use it on my components as we already have a project setup I cannot just clone and use the solution.

Also could I use the MMKV approach already merged?

Thanks in advance!

**Comment by frankcalise** [comment]
Hey @LeoRedin!

First, appreciate you using the Discussion board over opening an issue, so thank you for that :)

And yes absolutely, we'd actually prefer you tried it out and gave feedback on it! You basically can copy the PRs or follow the recipes over from the cookbook to work through the upgrade.

Go for it and let us know!

**Reply by LeoRedin** [reply]
No problem! That's what the discussion is for right? Discussing things haha. 

I'll then go over the PRs and migrate the code that I have here and test the approaches. I will set a reminder to give feedback as we develop and launch the new version of the app. I then ask is there anything else you guys want me to try? I can definitely sort some other things out. If not I'll just proceed with MMKV and the Theming.

**Comment by LeoRedin** [comment]
Howdy @frankcalise! I just wanted to share that the new Theming from v10 works beautifully in our project. We've been using it for the past months, easy to use, nice API and solves the main problem of switching themes. Great work!

Want me to close the discussion? Everything seems to be resolved 😄

**Reply by frankcalise** [reply]
Great to hear - thanks for the update!
